File: 04. Visual Basic/17. Arreglos/Arreglos.vb
Module Module1

    Sub Main()

        ' Arreglo dinámico
        Dim arreglo1(2) As Integer
        arreglo1(0) = 234
        arreglo1(1) = 235
        arreglo1(2) = 236

        ' Arreglo estático
        Dim arreglo2() As Integer = {234, 235, 236}

        ' Imprimir arreglo
        For i As Integer = 0 To arreglo1.Length - 1 Step 1
            Console.WriteLine(arreglo1(i))
        Next

        Console.ReadLine()

    End Sub

End Module

File: 02. C++/17. Arreglos/Arreglos.cpp
#include "iostream"
#include "string"
#include "stdlib.h"

using namespace std;

int main(){

	// Arreglo dinámico
    int* arreglo1 = new int[3];
    arreglo1[0] = 234;
    arreglo1[1] = 235;
    arreglo1[2] = 236;
    
    // Arreglo estático
    int arreglo2[3] = {234, 235, 236};
    
    // Imprimir arreglo
    for (int i = 0; i < 3; i++) {
        cout << arreglo1[i] << endl;
    }
   
	system("pause");
	return EXIT_SUCCESS;
	
}